### PR TITLE
CR: Added Sample Folder

### PR DESCRIPTION
**Pull request by lowesitoo**
- added sample folder

### CODE REVIEW

File: stacktrek-bootcamp-html/sample/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample Document</title>
+</head>
+<body>
+    <p>Is this approved?</p>
```
Comment (CanonMinds) [inline stacktrek-bootcamp-html/sample/index.html:10]: `p` tags could be encapsulated with `div`.

File: stacktrek-bootcamp-html/sample/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample Document</title>
```
Comment (CanonMinds) [inline stacktrek-bootcamp-html/sample/index.html:7]: Looks good, later on - you could add the CDN for font styles here in the `head` section

**Review by CanonMinds — APPROVED** [review]
This sample folder LGTM! :heavy_check_mark:  

Cool! :fireworks:  Congratulations!

File: stacktrek-bootcamp-html/sample/index.html (hunk 2 above)
Comment (lowesitoo) [reply stacktrek-bootcamp-html/sample/index.html:7]: Thanks! Will add next time!

File: stacktrek-bootcamp-html/sample/index.html (hunk 1 above)
Comment (lowesitoo) [reply stacktrek-bootcamp-html/sample/index.html:10]: This was updated. 🚀

**Pull request merged**